pendant page (#103)
farm-ng/amiga-dev-kit: website/docs/pendant/pendant.md

### Description
The pendant is a wired Manual joystick control interface with a rubberized backlit keypad that is  waterproof and rugged. This CANbus enabled controller is pluggable anywhere on the tractor communication backbone.

### How to connect

The pendant must be connect to CAN/Power accessory port which is location on the back of the dashboard panel. (please reference photo below)

![187535995-d4ad8c7c-05a0-482b-8be9-6b6fb6b87ac3](https://user-images.githubusercontent.com/64480560/210901594-6733b85d-292b-4468-9c65-3e275ed1c6f5.png)

### Pendant Controls

![IMG-9902](https://user-images.githubusercontent.com/64480560/211636387-fa11f5ea-59b0-4506-8005-d99ab21f90c6.jpg)

**Joystick** (omni directional stick that is used to control the amiga. This controls include, Acceleration, and turning left and right, reverse)

**Cruise(X)** – this will allow you to place your Amiga into cruise control. Press this button while the amiga is completely stopped. To activate this, press `cruise` then gently push forward the joystick to accelerate, pull back on joystick to decrease acceleration

> 

Reviewer: You can click cruise again to exit cruise control and slow to a stop.

:::caution
Pushing forward and pulling back on the joystick does not control speed, only acceleration and how much it will accelerate in relation to the set speed on the dashboard home tab.
:::

**Brake(O)**

> this is an emergency brake for the MCUs, pressing this will end cruise control and stop all motors

Reviewer: MCUs? What this means should be defined, but also I think we use MCU elsewhere to mean microcontroller (as with MCU kit)

:::caution
Pushing forward and pulling back on the joystick does not control speed, only acceleration and how much it will accelerate in relation to the set speed on the dashboard home tab.
:::

**Brake(O)**
This is an emergency brake for the microcontroller unit(MCU), pressing this will end cruise control and stop all motors

**PTO (Triangle)**
Power take-off (PTO) is a device that transfers an engine's mechanical power to another piece of equipment. A PTO allows the hosting energy source to transmit power to additional equipment that does not have its own engine or motor.

We use this to facilitate control from the pendant for addition equipment attached to the Amiga, such as the [3-point lift](https://farm-ng.com/products/cat-zero-3-point-lift-kit), [compost spreader](https://farm-ng.com/products/compost-spreader) or [cargo dump box](https://farm-ng.com/products/kit-cargo-dump-box)

**PAUSE(Square)**
Currently this button is a free button to be updated in future firmware releases.

**ARROWS** -  UP, Down, Left, Right {these are used to control your connect and registered PTO device)

>  Basic troubleshooting can found on our [FAQ page](https://amiga.farm-ng.com/docs/reference/faq)

Reviewer: ```suggestion
 Basic troubleshooting can found on our [FAQ page](/docs/reference/faq)
```
Use relative links

**Joystick** (omni directional stick that is used to control the amiga. This controls include, Acceleration, and turning left and right, reverse)

**Cruise(X)** – this will allow you to place your Amiga into cruise control. To activate this,while the amiga is completely stopped, press `cruise` then gently push forward the joystick to accelerate, pull back on joystick to decrease acceleration. You can press this button once more to exit cruise control and slow to a stop.

:::caution
Pushing forward and pulling back on the joystick does not control speed, only acceleration and how much it will accelerate in relation to the set speed on the dashboard home tab.
:::

**Brake(O)**

> This is an emergency brake for the microcontroller unit(MCU), pressing this will end cruise control and stop all motors

Reviewer: ```suggestion
This is a brake for the Amiga. Pressing this will stop all motors and exit cruise control mode (if active).
```
